addons/true-youtube-links/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "True forums YouTube links",
+  "name": "True forum YouTube links",
```

Comment:
"Forum" _does_ sound good here.

Reply:
We do need to be consistent, though.

I'm neutral on this one.

Reply:
I think "forums" because that's what's grammatically correct, and it's only one letter.